### ISSUE TITLE
Domain activation is Failed - [BRIEF DESCRIPTION]

### ISSUE THREAD

**Issue by Roger-Takeshita** [aws-amplify/amplify-console#1005]
**App Id**: d3cejlxi1c5gny
**Region**: ca-central-1
**Step**: Domain activation
**Status**: Failed

>**Note**: Do not include information that is sensitive in nature such as your domain name, company etc.

**Issue/question**
How can I associate my domain with a different repo? 
I recently removed workmail from my acc, but maybe I might have removed something else with the workmail because my domain is not showing on "AWS amplify > Domain management"

**Error message**
One or more of the CNAMEs you provided are already associated with a different resource.

**Additional information**
I tried to add a new repo to use with my domain, now I cannot associate with my domain. On "Route 53 > Hosted Zones > my_domain_name", I find some fields with the type "CNAME", should I delete them?

**Comment by hsspain** [comment]
@Roger-Takeshita You need to remove the domain association that your domain has with any other Amplify app.  It looks like it is already used in another account, possibly your workmail account.

**Comment by Roger-Takeshita** [comment]
@hsspain I think there is a bug on `Amplify Console > All app` after refreshing/accessing the page several times, out of nowhere my old app (app 1) showed up and then I was able to delete the old app (app 1). One thing strange is that my old app (app 1) showed up but my new app disappeared (app 2).
Then I created a new app (app 3) and successfully connected to my domain. Now my website is online, but when I accessed `Amplify Console > All app`, the app (app 2) that had disappeared showed up, and my brand new app (app 3) disappeared. And I was able to associate the same repo with two different apps (app 2 and app 3). I deleted the app 2 just in case, but my app 3 is not showing anywhere

<img width="938" alt="Screen Shot 2020-08-26 at 5 13 49 PM" src="https://user-images.githubusercontent.com/32579177/91357649-94d55480-e7bf-11ea-9d0a-de71a0b3927a.png">
